packages/css/src/highlight/highlight.scss

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+.amsterdam-highlight {
+  padding-block: var(--amsterdam-highlight-padding-block);
+  width: 100%;
+}
+
+.amsterdam-highlight--yellow {
+  background-color: var(--amsterdam-highlight-yellow-background-color);
+}
+
+.amsterdam-highlight--orange {
+  background-color: var(--amsterdam-highlight-orange-background-color);
+}
+
+.amsterdam-highlight--magenta {
+  background-color: var(--amsterdam-highlight-magenta-background-color);
+}
+
+.amsterdam-highlight--purple {
+  background-color: var(--amsterdam-highlight-purple-background-color);
+}
+
+.amsterdam-highlight--blue {
+  background-color: var(--amsterdam-highlight-blue-background-color);
+}
+
+.amsterdam-highlight--light-blue {
+  background-color: var(--amsterdam-highlight-light-blue-background-color);
+}
+
+.amsterdam-highlight--green {
+  background-color: var(--amsterdam-highlight-green-background-color);
+}
+
+.amsterdam-highlight--dark-green {
+  background-color: var(--amsterdam-highlight-dark-green-background-color);
+}
```

Comment:
Let’s sort lists like this alphabetically if no other order seems more appropriate.